internal/install/execution/output_parser.go

```diff
@@ -33,9 +33,13 @@ func (op *OutputParser) Metadata() map[string]string {
 	return nil
 }
 
-func (op *OutputParser) FailedRecipeOutput() string {
-	if val, ok := op.output["FailedRecipeOutput"]; ok {
-		return val.(string)
+func (op *OutputParser) LogCaptureEnabled() bool {
```

Comment:
@mbruzina small feedback about naming and consistency. It would be great to use a similar/identical naming between the var and its usage (reduces reader confusion when browsing across repositories). 
Also for boolean returned method, it usually reads better if the spelling is turned into a question, for example "IsLogCaptureEnabled". Otherwise this could be interpreted by a reader as "make the log capture enabled", or "log the enabled capture"